src/distilabel/steps/task/ultracm.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
+
+from jinja2 import Template
+from pydantic import PrivateAttr
+
+from distilabel.llm.base import LLM
+from distilabel.llm.vllm import vLLM
+from distilabel.steps.task.base import Task
+
+if TYPE_CHECKING:
+    from distilabel.steps.task.typing import ChatType
+
+
+_ULTRACM_TEMPLATE = """Given my answer to an instruction, your role is to provide specific and constructive feedback for me. You should find the best way for me to learn from your feedback and improve my performance.
```

Comment:
Could you follow the same approach as in #464? i.e. putting that under the `templates` directory with the name of the task? `ultra-cm.jinja2`?

Reply:
Sure, should I update the other tasks that write it as a string instead of the jinja template? (i.e. `ComplexityScorer`)

Reply:
Oops this is still missing, can you follow the same approach as in `core-refactor` with other tasks like i.e. `UltraFeedback`?